### DISCUSSION TITLE
"Every member, regardless of their role or contribution level, has an equal voice and is entitled to one vote in all decisions."

### DISCUSSION THREAD

**Discussion by yrodiere** [commonhaus/foundation#51] [General]
Taken from 5.1, General Decision Making:

> Every member, regardless of their role or contribution level, has an equal voice and is entitled
to one vote in all decisions.

I think the "all decisions" part kind of contradicts the fact that some topics are for the CFC to decide (and some even non-publicly).

I mean I get the idea, but as this is a legal document, this probably shouldn't be ambiguous... ?

**Comment by ebullient** [comment]
Yep. You're correct. We should, perhaps, just clarify this with some scoping?

- All CFC members have equal votes on council decisions.
- We haven't defined a Project Leads (group/board/name-of-body), but each Project Lead within that body has an equal vote on those decisions.
- All members have a voice, can raise issues (for all members, for all project leads, for the council), and are entitled to a vote in member decisions (which includes both elections and referendums)

Something like that?

**Reply by yrodiere** [reply]
Sounds good to me, thanks

**Reply by ebullient** [reply]
I think the first two have been covered in #52 ...

That revision to what members can do should be added to changes for #49 ..

Members are natural persons, and join as individuals. Each has a voice, can raise issues (for all members, for all project leads, for the council), and is entitled to a vote in member decisions (which includes both elections and referendums).

**Reply by ebullient** [reply]
I think we've resolved this one? Can we close it?

**Comment by yrodiere** [comment]
Yep, LGTM. Thanks.
